init.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
what is that file good for in the project root directory?

Reply:
removed. looks like its just for upstarting from sphinx-starter-pack